app/controllers/application_controller.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def defualt; end
+  def prevant; end
+  def centre; end
+  def theatre; end
```

Comment:
__[development][Misspell]__ <sub>posted by Sider</sub>

"theatre" is a misspelling of "theater"